ajax/libs/ali-oss/package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "ali-oss",
+  "filename": "videojs.ima.min.js",
```

```suggestion
  "filename": "aliyun-oss-sdk.min.js",
```

`videojs.ima.min.js` does not exist in the `dist` directory.

```
ls node_modules/ali-oss/dist

aliyun-oss-sdk.js     aliyun-oss-sdk.min.js
```